packages/shared/src/components/profile/UserShortInfo.tsx

```diff
@@ -0,0 +1,55 @@
+import classNames from 'classnames';
+import React, { ReactElement } from 'react';
+import { ProfileImageSize, ProfilePicture } from '../ProfilePicture';
+
+// reference: https://stackoverflow.com/a/54049872/5532217
+type AnyTag =
+  | string
+  | React.FunctionComponent<never>
+  | (new (props: never) => React.Component);
+
+type PropsOf<Tag> = Tag extends keyof JSX.IntrinsicElements
+  ? JSX.IntrinsicElements[Tag]
+  : Tag extends React.ComponentType<infer Props>
+  ? Props & JSX.IntrinsicAttributes
+  : never;
+
+interface UserShortInfoProps<Tag extends AnyTag> {
+  image: string;
+  name: string;
+  username: string;
+  bio?: string;
+  imageSize?: ProfileImageSize;
+  className?: string;
+  tag?: Tag;
+}
+
+export function UserShortInfo<Tag extends AnyTag>({
+  name,
+  username,
+  image,
+  bio,
+  imageSize = 'xlarge',
+  tag,
+  className,
+  ...props
+}: UserShortInfoProps<Tag> & PropsOf<Tag>): ReactElement {
+  const Element = (tag || 'a') as React.ElementType;
+
+  return (
+    <Element
+      {...props}
+      className={classNames(
+        'flex flex-row py-3 px-6 hover:bg-theme-hover',
+        className,
+      )}
+    >
+      <ProfilePicture user={{ username, image }} size={imageSize} />
```

Comment:
The following `{{ username, image }}` always creates a new object.
It's better to store it in `useMemo`